inc/Engine/Optimization/Minify/CSS/Minify.php

```diff
@@ -32,6 +33,19 @@ public function optimize( $html ) {
 		}
 
 		foreach ( $styles as $style ) {
+			foreach ( $this->get_excluded_externals() as $excluded ) {
```

Comment:
2 suggestions: 
- Initialize the excluded outside of the loops, that way the method is only called once
- Instead of a second loop, how about imploding the array into a string pattern, and using `preg_match()` to do the matching in the URL?

Reply:
Moved this check into `AbstractCSSOptimization::is_minify_excluded_file()`.